packages/remix-dev/compiler/compileBrowser.ts

```diff
@@ -82,6 +82,7 @@ const createEsbuildConfig = (
     entryPoints,
     outdir: config.assetsBuildDirectory,
     platform: "browser",
+    target: ['safari12'],
```

Comment:
If the goal is to support safari as documented, this should be `safari11` as that's the first version to support ES modules
<img width="1377" alt="Screenshot 2022-12-03 at 8 14 42 AM" src="https://user-images.githubusercontent.com/57655/205449090-98fc2d9f-7375-42ba-9e37-ef0be9c88874.png">

I created a ticket for this https://github.com/remix-run/remix/issues/4762

Reply:
the reason I pick `safari12` is because this error message

```
✘ [ERROR] Transforming async generator functions to the configured target environment ("safari11" + 1 override) is not supported yet

    node_modules/.pnpm/@web3-storage+multipart-parser@1.0.0/node_modules/@web3-storage/multipart-parser/esm/src/search.js:253:0:
      253 │ async function* arrayIterator(iter) {
          ╵ ~~~~~
```